dashboard/components/account-details/AwsAccountDetails.tsx

```diff
@@ -0,0 +1,184 @@
+import { ChangeEvent, ReactNode, useRef, useState } from 'react';
+import Folder2Icon from '../icons/Folder2Icon';
+import SelectInput from '../onboarding-wizard/SelectInput';
+import LabelledInput from '../onboarding-wizard/LabelledInput';
+import InputFileSelect from '../onboarding-wizard/InputFileSelect';
+import KeyIcon from '../icons/KeyIcon';
+import VariableIcon from '../icons/VariableIcon';
+import DocumentTextIcon from '../icons/DocumentTextIcon';
+import ShieldSecurityIcon from '../icons/ShieldSecurityIcon';
+import { CloudAccount } from '../cloud-account/hooks/useCloudAccounts/useCloudAccount';
+
+interface SelectOptions {
+  icon: ReactNode;
+  label: string;
+  value: string;
+}
+
+interface AwsAccountDetailsProps {
+  cloudAccountData: CloudAccount;
+  setCloudAccountData: (formData: CloudAccount) => void;
+}
+
+const options: SelectOptions[] = [
+  {
+    icon: <DocumentTextIcon />,
+    label: 'Credentials File',
+    value: 'credentials-file'
+  },
+  {
+    icon: <KeyIcon />,
+    label: 'Credentials keys',
+    value: 'credentials-keys'
+  },
+  {
+    icon: <VariableIcon />,
+    label: 'Environment Variables',
+    value: 'environment-variables'
+  },
+  {
+    icon: <ShieldSecurityIcon />,
+    label: 'IAM Instance Role',
+    value: 'iam-instance-role'
+  }
+];
+
+function AwsAccountDetails({
+  cloudAccountData,
+  setCloudAccountData
```

Comment:
```suggestion
function AwsAccountDetails({
  cloudAccountData,
  setCloudAccountData
```
above two props need to implement